app/models/event_log.rb

```diff
@@ -80,15 +80,17 @@ def send_to_splunk(*)
     return unless ENV['SPLUNK_EVENT_LOG_ENDPOINT_URL'] && ENV['SPLUNK_EVENT_LOG_ENDPOINT_HEC_TOKEN']
 
     event = {
```

Comment:
I wonder if it would be clearer to have it below like this instead:

```rb
request.body = { event: event }.to_json
```

Just because then we don't need the extra `event` key.

Reply:
Yup looks neater, done 👍